chef/cookbooks/crowbar/templates/default/puma.cfg.erb

```diff
@@ -0,0 +1,50 @@
+ROOT = File.expand_path("../../", __FILE__)
+ENVIRONMENT = ENV["CROWBAR_ENV"] || "production"
+
+THREADS = ENV["CROWBAR_THREADS"] || 1
+WORKERS = ENV["CROWBAR_WORKERS"] || 5
+
+LISTEN = "<%= @web_host %>"
+PORT = "<%= @web_port %>"
```

Comment:
As all other parts are changeable through env variables i would prefer that here as well :)